core/templates/dev/head/css/oppia.css

```diff
@@ -3457,6 +3457,12 @@ md-card.preview-conversation-skin-supplemental-card {
   color: #fff;
 }
 
+@media only screen and (max-width: 500px) {
+    button.btn.oppia-splash-button {
```

Comment:
Done. Pushing a commit and awaiting your confirmation on the above commit for another push